HPXMLtoOpenStudio/tests/test_validation.rb

```diff
@@ -2091,6 +2098,16 @@ def test_ruby_warning_messages
       when 'floor-or-ceiling2'
         hpxml, hpxml_bldg = _create_hpxml('base-foundation-unvented-crawlspace.xml')
         hpxml_bldg.floors[0].floor_or_ceiling = HPXML::FloorOrCeilingCeiling
+      when 'hpwh-installed-properly'
+        hpxml, hpxml_bldg = _create_hpxml('base-dhw-tank-heat-pump-containment-volume.xml')
+        hpxml_bldg.water_heating_systems[0].hpwh_installed_properly = true
+      when 'hpwh-small-containment-volume-without-backup-element'
+        hpxml, hpxml_bldg = _create_hpxml('base-dhw-tank-heat-pump-containment-volume.xml')
+        hpxml_bldg.water_heating_systems[0].backup_heating_capacity = 0.0
+        hpxml_bldg.water_heating_systems[0].hpwh_containment_volume = 250.0
```

Comment:
Unit test to verify that we throw warning that the adjustment may not be accurate when there's no backup heating